scripts/myvectorbench.py

```
@@ -0,0 +1,641 @@
+    """Run knn_queries ORDER-BY-distance queries against build_t; return timing metrics."""
+    n_queries = wp.get('knn_queries', 200)
+    print(f"  [knn_search] {n_queries} queries, dim={wp['dim']}")
+
+    rng = random.Random(99)
+    query_vectors = [vectors[rng.randint(0, len(vectors) - 1)] for _ in range(n_queries)]
+
+    latencies_ms = []
+    for q in query_vectors:
+        sql = (
+            f"SELECT id FROM bench.build_t"
+            f" ORDER BY myvector_row_distance(vec, {_vec_literal(q)}, 'L2') LIMIT 10;"
```

Ruff (0.15.13)

[error] Possible SQL injection vector through string-based query construction

(S608)